<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientUi</class>
 <widget class="QMainWindow" name="ClientUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientUi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>681</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>651</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="messageDisplay"/>
    </widget>
    <widget class="QLineEdit" name="messageInput">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>450</y>
       <width>551</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sendButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>450</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
